geode-core/src/main/java/org/apache/geode/management/internal/beans/DistributedSystemBridge.java

```diff
@@ -497,26 +499,15 @@ public DiskBackupStatus backupAllMembers(String targetDirPath, String baselineDi
         }
 
         DM dm = cache.getDistributionManager();
-        Set<PersistentID> missingMembers = MissingPersistentIDsRequest.send(dm);
```

Comment:
Why is this message no longer sent? What logic replaces it?

Reply:
None: this set does not end up actually being used for anything. It has members removed below from another set, but then the set is not used to do anything else, so it was either no longer/never needed or its purpose was never implemented.

Reply:
If the set isn't used, shouldn't the line change to:
```java
MissingPersistentIDsRequest.send(dm);
```

Reply:
From my reading of the code, there is no need to send the message at all. The only purpose of that message is to get back the information on missing members, but the result did not get used. From the code, it looked like they were trying to calculate the set of PersistentIDs that were missing from the backup, except that the backup itself was actually the right place to get that information and the initial attempt at gathering information here was abandoned, but not removed from the code. Do you know of a reason why the message would still need to be sent?